test/conformance/api/v1/volumes_test.go

```diff
@@ -457,12 +457,7 @@ func TestProjectedServiceAccountToken(t *testing.T) {
 	}
 	names.URL.Path = path.Join(names.URL.Path, tokenPath)
 	var parsesToken = func(resp *spoof.Response) (bool, error) {
-		jwtToken := string(resp.Body)
-		parser := &jwt.Parser{}
-		if _, _, err := parser.ParseUnverified(jwtToken, jwt.MapClaims{}); err != nil {
-			return false, err
-		}
-		return true, nil
+		return regexp.Match(`[A-Za-z0-9-_=]+\.[A-Za-z0-9-_=]+\.?[A-Za-z0-9-_.+/=]*`, resp.Body)
```

Comment:
I'd hoist an `isJWT` function and comment in there that we're trying to avoid a library dependency.  Is there a source for the regexp we can link to?

Reply:
Done!